lmd/response.go

```diff
@@ -488,7 +488,7 @@ func (res *Response) JSON() ([]byte, error) {
 		}
 	}
 	// append result row by row
-	if outputFormat == "wrapped_json" || outputFormat == "json" {
+	if outputFormat == "wrapped_json" || outputFormat == "json" || outputFormat == "python" {
```

Comment:
just remove the if clause, that block is valid for all supported output formats anyway. And if at some point we will support csv as well, this would go into a separate function.

Reply:
yah sure. Thanks